tailwind.config.js

```diff
@@ -15,9 +15,10 @@ module.exports = {
         '200px': '200px',
       },
       colors: {
-        'gray-tooltip': '#9e9e9e',
+        'tooltip-grey': '#9e9e9e',
         'ssw-red': '#cc4141',
         'ssw-grey': '#eee',
+        'light-grey': '#ccc',
         'ssw-black': '#333',
         underline: '#aaa',
       },
```

Comment:
We can change this to: (Just a note no real need to change for this amount of colours) 

```js
ssw: {
'red': '#cc4141',
etc... 
}
```

And it can be used the same way e.g. `ssw-red`